static/bidder-info/videoheroes.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: "http://point.contextualadv.com/?t=3&partner={{.PublisherID}}"
+maintainer:
+  email: "support@videoheroes.tv"
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - native
```

Comment:
I see that you support these media types, but dont see this reflected in your docs PR. Could you update it?

Reply:
Line 10, https://github.com/prebid/prebid.github.io/pull/4040/files